aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -287,9 +288,9 @@ public class Program
 
 ### Start and StopAsync
 
-[Start](/dotnet/api/microsoft.extensions.hosting.hostingabstractionshostextensions.start) starts the host synchronously.
+<xref:Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Start*> starts the host synchronously.
```

Comment:
This section's heading is confusing. It includes "StopAsync", but the section heading below does too. If I want to read about that method, which section do I look at?